### DISCUSSION TITLE
Easily browse through images in a stack

### DISCUSSION THREAD

**Discussion by djjudas21** [photoprism/photoprism#3427] [Ideas]
I am reviewing all of my stacked images. In some cases, the images in the stack appear to be identical, and I can delete one of them.

When I go into the File tab, I can click on either thumbnail to see a large version of the image, but it would be cool if I could move between the large versions with the arrow keys to easily compare if they are identical, or just similar. My "optical diff" :eyes: doesn't work well when I have to close one image and open the other. Thanks.

![image](https://github.com/photoprism/photoprism/assets/604595/0f33bd4a-0ada-49af-8545-25746644681f)

**Comment by graciousgrey** [comment]
At the moment the only way to "browse" through the images of a stack is described here:

https://docs.photoprism.app/user-guide/organize/stacks/#browse-related-files

I agree being able to browse related files when opening a photo from the files tab would be a nice enhancement.
